RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterCUDA.cc

```diff
@@ -89,7 +91,9 @@ SiPixelRawToClusterCUDA::SiPixelRawToClusterCUDA(const edm::ParameterSet& iConfi
           edm::ESInputTag("", iConfig.getParameter<std::string>("CablingMapLabel")))),
       isRun2_(iConfig.getParameter<bool>("isRun2")),
       includeErrors_(iConfig.getParameter<bool>("IncludeErrors")),
-      useQuality_(iConfig.getParameter<bool>("UseQualityInfo")) {
+      useQuality_(iConfig.getParameter<bool>("UseQualityInfo")),
+      clusterThresholds_{iConfig.getParameter<int32_t>("theGPUClusterThreshold_L1"),
+                         iConfig.getParameter<int32_t>("theGPUClusterThreshold")} {
```

Comment:
```suggestion
      clusterThresholds_{iConfig.getParameter<int32_t>("clusterThreshold_layer1"),
                         iConfig.getParameter<int32_t>("clusterThreshold_otherLayers")} {
```
to match the naming in SiPixelDigisClustersFromSoA.cc (consistency is important, IMHO)